addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -377,6 +413,18 @@ class MessageList extends Component {
         this.threadViewer.markComponentHintProcessed(hint);
     }
 
+    /**
+     * @private
+     */
+    _adjustScrollFromModel() {
```

Comment:
"from model"?

Reply:
Yeahh, from model. If you have a better name feel free, this suggestion was from @seb-odoo

Reply:
long explicit version: `_adjustScrollFromThreadCacheInitialScrollPosition`?

Reply:
Name's bad, as a result introduces a displeasing indirection. Also wording "initial" in scroll position field name is misleading.

All in all, I haven't found correct function/field naming/splitting, might need to make changes first to see it better...

Would rename "initial" scroll positions in a way to better capture the idea of saved scroll positions for recover, without being wrongly related to "initial" in some ways (looks like leftovers from handling scroll on mount... Which was bad and removed for this reason).

Maybe after this change, you may find a better naming for such a function, and certainly also make use of it in adjustment from change of thread cache, which feels natural.

Reply:
From my point of view it's just a name for a private method which could be changed later if you feel it's needed, I feel like it's a waste of time to rename all of this hoping that maybe a better name will come with it (and that was not really the intent of this PR by the way). So feel free to commit a change if you think the PR could not be merged like that :)